src/utils/event.ts

```diff
@@ -60,11 +60,21 @@ export const eventHappeningNow = (
 export const hasEventFinished = (event: VenueEvent) =>
   isAfter(Date.now(), eventEndTime(event));
 
-export const eventStartTime = (event: VenueEvent) =>
-  fromUnixTime(event.start_utc_seconds);
+export const eventStartTime = (event: VenueEvent) => {
+  if (!event?.start_utc_seconds) {
+    return new Date();
+  }
+
+  return fromUnixTime(event.start_utc_seconds);
+};
 
-export const eventEndTime = (event: VenueEvent) =>
-  addMinutes(eventStartTime(event), event.duration_minutes);
+export const eventEndTime = (event: VenueEvent) => {
```

Comment:
had some events with missing details, so had to add this guard to prevent timing page from breaking